src/sagas/identity.js

```diff
@@ -170,14 +185,27 @@ function * handleAddNewCategory(action) {
       name: action.payload.value,
       desc: '',
       identity: selectedIdentityId,
-      claims: [],
     },
   };
 
   yield put(setNewCategory(newCategory[newCategoryId]));
   yield call(updateClaimCategoryStorage);
 }
 
+function * handleMoveClaims(action) {
```

Comment:
Start thinking about how the API will be integrated an error handling, so your code can anticipate it easily.